.github/workflows/nightly.yml

```
@@ -35,13 +34,12 @@ jobs:
           systemd-devel
-      - uses: dtolnay/rust-toolchain@master
-        with:
-          components: ${{ matrix.components }}
-          toolchain: ${{ matrix.toolchain }}
       - uses: actions/checkout@v6
         with:
           persist-credentials: false
+      - uses: actions-rust-lang/setup-rust-toolchain@v1
+        with:
+          components: ${{ matrix.components }}
       - name: Test ${{ matrix.task }} on ${{ matrix.toolchain }} toolchain
```

actionlint (1.7.12)

[error] property "toolchain" is not defined in object type {components: string; task: string}

(expression)